wp-web-push/wp-web-push-admin.php

```diff
@@ -26,34 +28,34 @@ public function options() {
       } else if ($_POST['webpush_title'] === 'custom') {
         update_option('webpush_title', $_POST['webpush_title_custom']);
       } else {
-        wp_die('Wrong value for the Notification Title');
+        wp_die(__('Wrong value for the Notification Title', 'wpwebpush'));
       }
 ?>
-<div class="updated"><p><strong>Settings saved.</strong></p></div>
+<div class="updated"><p><strong><?php _e('Settings saved'); ?></strong></p></div>
 <?php
     }
 ?>
 
 <div class="wrap">
-<h2>Web Push</h2>
+<h2><?php _e('Web Push', 'wpwebpush'); ?></h2>
 
 <form method="post" action="">
 <table class="form-table">
 
 <tr>
-<th scope="row">Notification Title</th>
+<th scope="row"><?php _e('Notification Title', 'wpwebpush'); ?></th>
 <td>
 <fieldset>
-<label><input type="radio" name="webpush_title" value="blog_title" <?php echo $title_option === 'blog_title' ? 'checked' : '' ?> /> Use the Site Title: <b><?php echo get_bloginfo('name'); ?></b></label><br />
-<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> Custom:</label>
-<input type="text" name="webpush_title_custom" value="<?php echo $title_option !== 'blog_title' ? $title_option : 'Write here the title' ?>" class="long-text" />
+<label><input type="radio" name="webpush_title" value="blog_title" <?php echo $title_option === 'blog_title' ? 'checked' : '' ?> /> <?php _e('Use the Site Title', 'wpwebpush'); ?>: <b><?php echo get_bloginfo('name'); ?></b></label><br />
+<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> <?php _e('Custom', 'wpwebpush'); ?>:</label>
```

Comment:
I'm not sure how this works, but probably some of these strings will already be localized in WordPress (for example "Custom", "Save Changes", "Saved settings", etc.).
We can figure out how to deal with those later (we can re-localize them all, or we can use the WordPress localized strings).